strawberry/starlite/controller.py

```diff
@@ -0,0 +1,399 @@
+"""Starlite integration for strawberry-graphql."""
+import json
```

Comment:
For better performance import `encode_json` and `decode_json` from `starlite.utils.serialization`